.pre-commit-config.yaml

```diff
@@ -1,5 +1,21 @@
 ---
 repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks.git
+  rev: v2.4.0
+  hooks:
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+    exclude: |
+      (?x)(
+        ^test/(with-skip-tag-id|unicode).yml|
+        ^examples/example.yml
+      )
```

Comment:
```suggestion
    exclude: >
      (?x)^(
        test/(with-skip-tag-id|unicode).yml|
        examples/example.yml
      )$
```

Reply:
I am really not confident that his would work correctly based on how ">" yaml joining is working. I know for sure that the proposed one works well. Why to waste another 2-3h on a variation?